.circleci/config.yml

```diff
@@ -101,6 +101,7 @@ jobs:
       CIRCLE_TEST_REPORTS: /tmp/test-reports
       CIRCLE_ARTIFACTS: /tmp/artifacts
     steps:
+      - setup_remote_docker # required for using docker commands: https://circleci.com/docs/building-docker-images/#introduction
```

Comment:
Do we still need this? I think we wanted to test something with antithesis, but not sure if you still want this for prod